ckanext/csrf_filter/plugin.py

```diff
@@ -62,6 +65,11 @@ def login(self):
         request.get_environ()['__no_cache__'] = True
         return None
 
+    # ITemplateHelpers
+
+    def get_helpers(self):
+        return {'csrf_token_field': h.csrf_token_field}
```

Comment:
Since this helper depends on Flask, it should only be applied if we're on CKAN 2.9+

Reply:
Handled pylons and flask with `is_flask_request` from common. As it is listed as a deprecated method now, I have put it in a try/catch for some future proofing